raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -155,6 +155,10 @@ describe('Raiden', () => {
   const txBlock = 42;
   const txHash = makeHash();
   const transferAmount = 42;
+  const id = 10;
```

Comment:
is this the channel id? if so please rename it.

Reply:
yes this is the channel Id I will rename it.